functions-and-scope.js

```diff
@@ -13,6 +15,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorg ik ervoor dat dit ook werkt wanneer de array 100 entries bevat?
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
+let highGrades = 0;
+for (let i = 0; i < grades.length; i ++) {
+   if (grades[i] >= 8) {
+       const isCumLaude  = grades[i];
+       highGrades = highGrades + 1;
```

Comment:
Deze kun je nog iets korter schrijven
```suggestion
       highGrades++;
```